### PR TITLE
Adjust OuiSimplifiedBreadcrumbs alignment, colors, and tailing slash

### PR DESCRIPTION
**Pull request by AMoo-Miki**
### Description
Add methods to control the appearance of trailing and last breadcrumbs in OuiSimplifiedBreadcrumbs

Also:
* Adjust alignments, colors, and separator in OuiSimplifiedBreadcrumbs
* Add documentation for OuiSimplifiedBreadcrumbs
* Add CSS breakpoints to OuiBreakpointSize
* Remove the unwanted OuiBreadcrumbs focus rectangle
* Correctly color the OuiBreadcrumbs when focused
* Display the last breadcrumb in a nested breadcrumb as a normal breadcrumb

### Check List
- [ ] New functionality includes testing.
- [ ] New functionality has been documented.
- [ ] All tests pass
  - [ ] `yarn lint`
  - [ ] `yarn test-unit`
- [X] Update [CHANGELOG.md](./../CHANGELOG.md)
- [X] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/oui/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: src/components/breadcrumbs/_breadcrumbs_simplified.scss [hunk 1]
```diff
@@ -77,9 +86,8 @@
...
   background: $ouiColorLightShade;
   color: $ouiColorDisabledText;
-}
 
-.euiLink.euiSimplifiedBreadcrumb {
-  line-height: inherit;
-  font-weight: inherit;
-}
\ No newline at end of file
+  &:last-child {
```
Comment (virajsanghvi) [inline src/components/breadcrumbs/_breadcrumbs_simplified.scss:90]: does it still show up when the last child is not a leaf node?

Comment (AMoo-Miki) [reply src/components/breadcrumbs/_breadcrumbs_simplified.scss:90]: need to address this

**Review by virajsanghvi — COMMENTED** [review]
Do you know why this doesn't match oui website?

your changes: image

https://oui.opensearch.org/1.12/#/navigation/breadcrumbs: 
image

Also, are failures related?

**Review by virajsanghvi — COMMENTED** [review]
Change looks good, although looks like there are test failures

**Comment by AMoo-Miki** [comment]
> Do you know why this doesn't match oui website?

Two changes impact this:
1. The new breakpoints fine-tune the number of crumbs shown
2. A fix to breakpoint identification corrects the logic

**Review by virajsanghvi — APPROVED** [review]

**Pull request merged**